src/formik.tsx

```diff
@@ -61,6 +61,8 @@ export interface FormikConfig<
   validateOnChange?: boolean;
   /** Tells Formik to validate the form on each input's onBlur event */
   validateOnBlur?: boolean;
+  /** Tell Formik if initial form values are valid or not on first render */
+  isInitialValid?: (props: Props) => boolean;
```

Comment:
I couldn't get typescript to allow me to make this a`boolean | (props: Props) => boolean` unless we make it a required option (which we don't want). Thoughts? Am I missing something?